staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -4528,6 +4536,30 @@ type PodSchedulingGate struct {
 	Name string `json:"name" protobuf:"bytes,1,opt,name=name"`
 }
 
+// WorkloadReference identifies the Workload object and PodGroup membership
+// that a Pod belongs to. The scheduler uses this information to enforce
+// gang scheduling semantics.
```

Comment:
Let's start with already more generic comment.

The scheduler uses this information to apply workload-aware scheduling semantics.